### PR TITLE
Bare metal image

### PR DESCRIPTION
**Pull request by gaohannk**
This is bare metal image build workflow

### CODE REVIEW

File: daisy_workflows/build-publish/bare_metal/rhel_8.wf.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "source_image_project": "bct-prod-images",
+    "build_date": "${DATE}",
+    "workflow_root": {"Value": "/workflows", "Description": "Root of github workflows, defaults to /workflows in the container."},
+    "gcs_url": {"Required": true, "Description": "The GCS path that image raw file exported to."}
+  },
+  "Steps": {
+    "build-bare-metal": {
+      "Timeout": "70m",
+      "IncludeWorkflow": {
+        "Path": "bare_metal_linux.wf.json",
```
Comment (zmarano) [inline daisy_workflows/build-publish/bare_metal/rhel_8.wf.json:16]: "Path": "./bare_metal_linux.wf.json",

File: daisy_workflows/build-publish/bare_metal/rhel_7.wf.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "source_image_project": "bct-prod-images",
+    "build_date": "${DATE}",
+    "workflow_root": {"Value": "/workflows", "Description": "Root of github workflows, defaults to /workflows in the container."},
+    "gcs_url": {"Required": true, "Description": "The GCS path that image raw file exported to."}
+  },
+  "Steps": {
+    "build-bare-metal": {
+      "Timeout": "70m",
+      "IncludeWorkflow": {
+        "Path": "bare_metal_linux.wf.json",
```
Comment (zmarano) [inline daisy_workflows/build-publish/bare_metal/rhel_7.wf.json:16]: "Path": "./bare_metal_linux.wf.json",

**Review by zmarano — CHANGES_REQUESTED** [review]
I think we should keep everything the same. The build workflows should be in daisy_workflows/image_build and these top level called workflows in build-publish for now. Otherwise this will get too confusing.

**Comment by gaohannk** [comment]
> I think we should keep everything the same. The build workflows should be in daisy_workflows/image_build and these top level called workflows in build-publish for now. Otherwise this will get too confusing.

Great Catch. We could include the top level in `build-publish` and the actually build workflow in `daisy_workflows/image_build`.

File: daisy_workflows/build-publish/bare_metal/rhel_8.wf.json (hunk 1 above)
Comment (gaohannk) [reply daisy_workflows/build-publish/bare_metal/rhel_8.wf.json:16]: done

File: daisy_workflows/build-publish/bare_metal/rhel_7.wf.json (hunk 2 above)
Comment (gaohannk) [reply daisy_workflows/build-publish/bare_metal/rhel_7.wf.json:16]: done.

Comment (gaohannk) [reply daisy_workflows/build-publish/bare_metal/rhel_7.wf.json]: This is not validate after move wf into image_build folder

File: daisy_workflows/build-publish/bare_metal/rhel_8.wf.json (hunk 1 above)
Comment (gaohannk) [reply daisy_workflows/build-publish/bare_metal/rhel_8.wf.json]: This is not validate after move wf into image_build folder

**Review by zmarano — APPROVED** [review]

**Pull request merged**